lib/task/tools/expireDataTask.class.php

```diff
@@ -153,4 +202,32 @@ private function clipboardExpireData($options)
 
     return $deletedCount;
   }
+
+  private function jobExpireData($options)
+  {
+    // Assemble criteria
+    $criteria = new Criteria;
+
+    if (isset($options['older-than']))
+    {
+      $criteria->add(QubitJob::CREATED_AT, $options['older-than'], Criteria::LESS_THAN);
+    }
+
+    // Delete jobs and save items
+    $deletedCount = 0;
+
+    foreach(QubitJob::get($criteria) as $job)
+    {
+      // Finding aids won't be deleted because the job doesn't store the path to them
```

Comment:
_Not_ deleting finding aids is the desired behavior, right?  I think it would be helpful to make this explicit in the comment. :)